src/settings/settings.css

```diff
@@ -1,123 +1,311 @@
+/* Root Variables */
 :root {
-    --background-color: #F0F0F0;
+    /* General vars */
+    --background-color: #f0f0f0;
     --form-background-color: white;
     --box-shadow-color: rgba(0, 0, 0, 0.1);
     --border-color: #ccc;
     --button-background-color: #003366;
-    --button-hover-background-color: #0056B3;
+    --button-hover-background-color: #0056b3;
+    --font: "Lato", sans-serif;
+    --height: 100vh;
+
+    /* Form vars */
+    --form-padding: 1.5em;
+    --form-border-radius: 5px;
+    --form-width: 400px;
+    --form-margin-top: 1em;
+    --form-box-shadow: 0 0 10px var(--box-shadow-color);
+
+    /* Label vars */
     --label-color: #003366;
+    --label-display: block;
+    --label-margin-top: 1em;
+    --label-font-weight: bold;
+    --label-text-align: left;
+    --label-font-size: 1em;
+
+    /* Input vars */
+    --input-padding: 0.5em;
+    --input-margin-top: 0.5em;
+    --input-margin-bottom: 1em;
+    --input-border-radius: 3px;
+    --input-border-width: 1px;
+    --input-box-sizing: border-box;
+    --input-width-calc: calc(100% - 10px);
+
+    /* Button vars */
+    --button-padding: 0.5em 1em;
+    --button-color: white;
+    --button-border: none;
+    --button-border-radius: 3px;
+    --button-margin-top: 1em;
+    --button-cursor: pointer;
+    --button-font-size: 1em;
+
+    /* Header vars */
+    --header-margin-bottom: 1em;
+    --header-font-size: 2em;
+    --header-font-weight: bold;
```

Comment:
Could you change --header-font-weight to normal to maintain consistency among pages.